<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="tj2_romi_odom" pkg="tj2_romi_odom" type="tj2_romi_odom_node" output="screen" required="true">
        <param name="min_angular_speed" value="0.7"/>
        <param name="max_angular_speed" value="7.0"/>
        <param name="min_linear_speed" value="0.05"/>
        <param name="max_linear_speed" value="0.33"/>
        <param name="zero_speed_epsilon" value="0.01"/>
        <param name="publish_odom_tf" value="false"/>
    </node>
</launch>
